func/code.func

```diff
@@ -92,105 +93,120 @@ _ unpack_order(slice order) inline_ref {
 
   slice sender = cs~load_msg_addr();
 
-  (slice stats, cell orders, slice owner1, slice owner2, slice echo) = unpack_state();
+  (slice stats, cell orders, slice owner, slice ecvrf) = unpack_state();
+  ;; subscribe to ecvrf 
+  send_raw_message(
+    begin_cell()
+    .store_uint(0x18, 6)
+    .store_slice(ecvrf)
+    .store_coins(0)
```

Comment:
Attempt to send 0 TON + gas fees will only send message, but not let the coordinator contract process it and add the lottery contract into broadcast group.